packages/editor/__tests__/operations/component.spec.ts

```diff
@@ -48,6 +49,35 @@ describe('Component operations', () => {
     ).toBe('stack5_copy0');
   });
 
+  it(`Paste toplevel component with children and the children's slot trait is correct`, () => {
+    const appModel = new AppModel(
+      PasteComponentWithChildrenSchema.spec.components,
+      registry
+    );
+
+    const pasteComponent = [
+      appModel.getComponentById('stack3' as ComponentId)!.toSchema(),
+      appModel.getComponentById('stack5' as ComponentId)!.toSchema(),
+      appModel.getComponentById('text6' as ComponentId)!.toSchema(),
+    ];
+    const operation = genOperation(registry, 'pasteComponent', {
+      parentId: RootId,
+      slot: '',
```

Comment:
`slot` is empty?

Reply:
Yes, because the copy is of the top-level component